modules/special-rules/pages/tyranid-objectives.adoc

```diff
@@ -0,0 +1,16 @@
+= Tyranid Objectives
+
+Always ready for a krumpin, Orks will often charge into melee before the enemy has even realised that the Orks are close enough to charge them!
+
+---
+
+Apply these rules to Tyranid armies:
+
+* The army does not use objectives, regardless of scenario.
+* If an enemy detachment is wiped out, add its morale value to the Tyranid player's army morale in the Rally phase. [either/or]
+* If an enemy detachment is wiped out, in the Rally phase the Tyranid player's army morale goes up by the detachment's morale value. [either/or]
```

Comment:
```suggestion
```
Again, nasty old convoluted passive case. Bad practice, better ways to do it. GW and others use it waaaay too much.

Reply:
If we use the raise/lower version:
```suggestion
* If an enemy detachment is wiped out, raise the Tyranid player's army morale by the detachment's morale value in the Rally phase.
```

Reply:
I don't think you understand what I mean about [passive case](https://epicremastered.com/contributors-guide/a-to-z-of-style.html#_p).
Raise/lower is a different issue (this is active where 'goes up' is not and where 'rises' would've been equivalent — but both are a bit figurative, 'add/remove' is better per other conversation).
There's also third person in the original construct.

My suggestion replaces passive/3rd and with active/2nd. It also replaces the figurative 'goes up' with plain 'add'.

Reply:
Marking as resolved as this was an alternative bullet point option for the one that is being committed above.